Add Python packages requirements

I've been trying to add your hook to [my dotfiles](https://github.com/nagromc/dotfiles), but I came across some missing Python packages. First, [PyYAML](http://pyyaml.org/), then [pygit2](http://www.pygit2.org/).

Wouldn't be a good idea to add the needed Python packages in README.md? Even better: a single-liner to install them for each system (e.g. for Debian: `sudo apt-get install python3-yaml python3-pygit2 ...`)

Note: on Debian, I stumbled upon an issue where [pygit2 is not included in the stable version](https://packages.debian.org/search?keywords=pygit2). So, I guess it would be difficult to make it a single-liner.

Well, OK. Here is what I've tried to install `pygit2` on Debian Jessie (the current stable):
- `sudo apt-get install libffi-dev libgit2-dev`
- `pip install pygit2`

`pip` logs:

```
[...] some logs before

running build_ext

generating cffi module 'build/temp.linux-x86_64-2.7/pygit2._libgit2.c'

creating build/temp.linux-x86_64-2.7

building '_pygit2' extension

creating build/temp.linux-x86_64-2.7/src

x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -D_FORTIFY_SOURCE=2 -g -fst
ack-protector-strong -Wformat -Werror=format-security -fPIC -I/usr/local/include -I/usr/include/python2.7 -c src/blob.c -o build
/temp.linux-x86_64-2.7/src/blob.o

In file included from src/blob.h:34:0,

                 from src/blob.c:30:

src/types.h:36:2: error: #error You need a compatible libgit2 version (v0.24.x)

 #error You need a compatible libgit2 version (v0.24.x)

  ^

error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

Unfortunately, [`libgit2` on Debian Jessie](https://packages.debian.org/jessie/libgit2-dev) is only in version 0.21.1 whereas `pygit2` requires `libgit2` v0.24.x. So I cannot install `pygit2`.

--- a/README.md
+++ b/README.md
@@ -3,6 +3,15 @@
 - A Dotbot configuration file in that repository
 - Python 3
 
+#### Dependencies installation
+
+##### Debian Jessie
+
+On Debian Jessie, run the following commands:
+
+    # apt-get install libgit2-dev python3-cffi python3-yaml
+    # pip3 install -r requirements.txt
+
 
 ### Instructions
 Save `pre-commit` and `prepare-commit-msg` to `.git/hooks` in your dotfiles directory. Alternatively, use a Git hook manager like [git-hooks](https://github.com/git-hooks/git-hooks).

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,3 @@
-pygit2
+pygit2==0.21.1
 pyyaml
 unidiff

